gemini/prompts/prompt_optimizer/vertex_ai_prompt_optimizer_sdk.ipynb

```diff
@@ -361,7 +361,9 @@
         "-   `Storage Object Admin` to read and write to your GCS bucket.\n",
```

Comment:
<p>FYI when I tried this in colab it failed complaining about conditions.</p><p>Adding --condition=None fixed it.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/1221/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>

Reply:
done